rllib/core/rl_trainer/scaling_config.py

```diff
@@ -14,8 +14,11 @@ class TrainerScalingConfig:
         num_gpus_per_worker: The number of GPUs to allocate per worker. If
             num_workers=0, any number greater than 0 will run the training on a single
             GPU. A value of zero will run the training on a single CPU.
+        local_gpu_id: if num_gpus_per_worker > 0, and num_workers=0, then this gpu id
+            will be used for training.
```

Comment:
explain what the default of zero exacly means with an example. for example in torch it's `cuda:0`